crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -876,11 +876,6 @@ async fn test_app_status_states() {
         .unwrap();
     assert_matches!(get_status().await, AppInfoStatus::Disabled { .. });
 
-    // DISABLED  --start->  DISABLED
```

Comment:
`start_app` didn't change enable app status when disabled. `enable_app` is used right after the next command, so there's no replacement for this part of the test.